### PR TITLE
Fix overflowing search hinttext on small screens

### PR DESCRIPTION
**Pull request by Argn0**
fixes https://github.com/odota/web/issues/1598

### CODE REVIEW

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -137,13 +137,17 @@ class Header extends React.Component {
       </VerticalAlignToolbar>
     );
 
-    const SearchGroup = () => (
+    const SearchGroup = ({ small }) => (
```
Comment (howardchung) [inline src/components/Header/Header.jsx]: are you actually using this prop?

Comment (Argn0) [reply src/components/Header/Header.jsx]: yes. SearchGroup is a parent of SearchForm. I use it in SearchForm.jsx

Comment (Argn0) [reply src/components/Header/Header.jsx]: or wait let me double check....

Comment (Argn0) [reply src/components/Header/Header.jsx]: ok it goes Redux Store -> Header -> SearchGroup > SearchForm

Comment (Argn0) [reply src/components/Header/Header.jsx]: this is probably bad. I should not do this :) I'm gonna get the prop in SearchForm directly.

**Pull request merged**